unix/xdebug

```diff
@@ -1,11 +1,23 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+if [ -f "${HOME}/.config/xdebug-wrapper" ]; then
+    . "${HOME}/.config/xdebug-wrapper"
+fi
 
 # Update according to your needs
-XDEBUG_FILE=php_xdebug-2.4.0-7.0-vc14.dll
-XDEBUG_PATH=/path/to/
-PHP_BINARY=/usr/bin/php
-XDEBUG_IDEKEY=PHPSTORM
+PHP_BINARY=${PHP_BINARY:="php"}
```

Comment:
The ```/usr/bin/env``` is necessary only in shebang line because kernel does not search PATH environment variable as bash does.